pkg/controller/serviceaccount/tokens_controller.go

```diff
@@ -332,7 +397,8 @@ func (e *TokensController) createSecret(serviceAccount *api.ServiceAccount) erro
 
 	// Save the secret
 	if createdToken, err := e.client.Core().Secrets(serviceAccount.Namespace).Create(secret); err != nil {
-		return err
+		// retriable error
+		return true, err
 	} else {
```

Comment:
if you mean remove the `else`, I'd rather keep `createdToken` and `err` scoped to this block